### ISSUE TITLE
Invalid start_url with revision in v3.3.0

### ISSUE THREAD

**Issue by andrewspy** [nuxt-community/pwa-module#403]
v3.3.0 added random per-build revision to `sw.js` with invalid URL.

Actual start URL:  `/start_url&revision=xxxxxxxx`
Expected start URL: `/start_url?revision=xxxxxxxx`

Error in chrome as follow:

```text
workbox-precaching.prod.js:1 Uncaught (in promise) bad-precaching-response: bad-precaching-response :: [{"url":"http://localhost:3000/start_url&revision=xxxxxxxx","status":404}]
```

**Comment by pi0** [comment]
Thanks for report. Fixed in 3.3.1. BTW any reason customizing it?

**Comment by andrewspy** [comment]
@pi0 It's working as expected now. Thanks!

In my use case, I need to change the route depending on the type of user (i.e. admin vs user).
